packages/example-getting-started/docs/scaffolding.md

```diff
@@ -0,0 +1,95 @@
+  application.ts
+  index.ts
+  sequence.ts
+test/
+  README.md
+  mocha.opts
+  ping.controller.test.ts
+node_modules/
+  ***
+LICENSE
+README.md
+index.js
+index.ts
+package.json
+tsconfig.json
+tslint.build.json
+tslint.json
+```
+
+| File | Purpose |
+|======|=========|
+| index.ts | Allows importing contents of the `src` folder (for use elsewhere) |
+| index.js | Top-level wireup for execution of the application. |
+| package.json | Your application's package manifest. See [package.json](https://docs.npmjs.com/files/package.json) for details. |
+| tsconfig.json | The TypeScript project configuration. See [tsconfig.json](http://www.typescriptlang.org/docs/handbook/tsconfig-json.html) for details. |
+| tslint.json | [TSLint configuration](https://palantir.github.io/tslint/usage/tslint-json/) |
+| tslint.build.json | [TSLint configuration (build only)](https://palantir.github.io/tslint/usage/tslint-json/) |
+When you're ready, we'll begin laying the groundwork for our application by
+making use of the [Legacy Juggler](juggler.md).
+| README.md | The Markdown-based README generated for your application. |
+| LICENSE | A copy of the MIT license. If you do not wish to use this license,
+please delete this file. |
+| src/application.ts | The application class, which extends [`RestApplication`](http://apidocs.strongloop.com/@loopback%2frest/#RestApplication) by default. This is the root of your application, and is where your application will be configured. |
+| src/index.ts | The starting point of your microservice. This file creates an instance of your application, runs the booter, then attempts to start the [`RestServer`](http://apidocs.strongloop.com/@loopback%2frest/#RestServer) instance bound to the application. |
+| src/sequence.ts | An extension of the [Sequence](http://loopback.io/doc/en/lb4/Sequence.html) class used to define the set of actions to take during a REST request/response. |
+| src/controllers/README.md | Provides information about the controller directory, how to generate new controllers, and where to find more information. |
+| src/controllers/ping.controller.ts | A basic controller that responds to GET requests at `/ping`. |
+| src/datasources/README.md | Provides information about the datasources directory, how to generate new datasources, and where to find more information. |
+| src/models/README.md | Provides information about the datasources directory, how to generate new datasources, and where to find more information. |
+| src/repositories/README.md | Provides information about the repositories directory, how to generate new repositories, and where to find more information. |
+| test/README.md | Please place your tests in this folder. |
+| test/mocha.opts | [Mocha](https://mochajs.org/) configuration for running your application's tests. |
+| test/ping.controller.test.ts | An example test to go with the ping controller in `src/controllers`. |
+
+<!--
+kjdelisle: Why are we automatically generating MIT license files for
```

Comment:
+1 to this. But I think this comment should be removed and a new issue should be created to capture this with a note to update this tutorial once CLI has been fixed.

Reply:
https://github.com/strongloop/loopback-next/issues/1079 is now open for this problem, and I'll push the changes to remove the comment soon.